internal/driver/glfw/window.go

```diff
@@ -857,7 +873,26 @@ func (w *window) keyPressed(viewport *glfw.Window, key glfw.Key, scancode int, a
 	keyEvent := &fyne.KeyEvent{Name: keyName}
 	keyDesktopModifier := desktopModifier(mods)
 
-	if keyName == fyne.KeyTab {
+	// Activate the menu when the Alt key is pressed
+	focused := w.canvas.Focused()
+	isMenu := false
+	menu, ok := w.canvas.menu.(*MenuBar)
+	if ok {
+		isMenu = menu.active
+		if !isMenu && action == glfw.Press && (key == glfw.KeyLeftAlt || key == glfw.KeyRightAlt) &&
+			keyDesktopModifier == desktop.AltModifier {
+			mbi := menu.Items[0].(*menuBarItem)
+			mbi.Tapped(nil)
+		}
+	}
+	if isMenu {
+		if action == glfw.Release {
+			menu.HandleKey(keyName)
+		}
+		return
+	}
+
+	if keyName == fyne.KeyTab && action != glfw.Release && !isMenu {
```

Comment:
The release check appears duplicating 2 liones below?

Reply:
Correct. Will fix.